test/cli/cli.go

```diff
@@ -131,16 +133,23 @@ func getAddCommand() *cobra.Command {
 // getAddSimulatorCommand returns a cobra command for deploying a device simulator
 func getAddSimulatorCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "simulator [name]",
+		Use:   "simulator [type] [name]",
```

Comment:
can this type be placed in the SimulatorConfig object ? thus removing many changes from the API ? If instead this breaks the point of SimulatorConfig I would move the type string to the end of the constructor arguments for consistency reasons and to leave the device name as first element. cosmetic, but enhances readability

Reply:
Agreed.

Reply:
agreed. Will do.

Reply:
Should this be called `device` instead of `simulator`? Or we could do:
```
onit add simulator ...
onit add stratum ...
```

Or the type can be a flag with a default value:
```
onit add device --type=simulator my-device
onit add device --type=stratum my-device 
```

Just change the `simulator` command to `device`, add an alias to `simulator` to avoid breakage, and default `type` to `simulator` or `onos` or whatever it should be.

Reply:
OK. I think the second one is better (i.e. onit add device .....) . I am working on it.

Reply:
agree

Reply:
I like the device better, indeed

Reply:
> We need to update all the commands:
> 
> ```
> onit delete device ...
> onit get devices ...
> ```
> 
> As well as the autocompletion code for devices.

OK. I will take care of them.